### ISSUE TITLE
Make this an organization page

### ISSUE THREAD

**Issue by ascott1** [eregs/eregs.github.io#19]
We could rename this repo so that the repo is **eregs.github.io** rather than **eregs.github.io/eRegulations**. Any objections?

**Comment by cmc333333** [comment]
Makes sense to me -- we'll just need to add HTML redirects as I'm sure that (and the cfpb version) have been referenced a million times.

**Comment by ascott1** [comment]
I _think_ GitHub is fancy and handles the redirects for us, but we should confirm that first.

**Comment by cmc333333** [comment]
The last time we tried this (when moving this repo to `cfpb` _from_ `eregs`), it did not handle gh-pages redirects. Pulling/pushing repos redirects and accessing the repo through the Github UI redirects, but not gh-pages.

**Comment by cmc333333** [comment]
@ascott1 How about this: unless you get to this first, I'll go ahead and rename this to the organization repo by next Wednesday. In that process, if we see that redirects aren't occurring, we can immediately push a duplicate of the content to `eregs/eRegulations`. We'll add redirects the following week if need be.

Sound good?

**Comment by ascott1** [comment]
I think that's a great plan. Thanks @cmc333333!

**Comment by cmc333333** [comment]
And by Wednesday, I mean Friday. Late, late Friday.

**Comment by cmc333333** [comment]
And by late, late Friday, I mean early, early Thursday.

Renamed this to be an organization repo. Unfortunately, github pages do not redirect, so I've also pushed the repo contents to eregs/eRegulations. We need to add HTML redirects to that repo before we can close this issue.
